scripts/setup-emlinux

```diff
@@ -68,4 +68,4 @@ clean_up
 if [ $_ERROR -ne 0 ]; then
     echo "ERROR: an error occured during setup ($_ERROR)"
 fi
-return $_ERROR
+exit $_ERROR
```

Comment:
Why? This script is supposed to be used by "source" command. If we use "exit" here instead of "return", this script causes your shell (callee of this script) to exit.

Reply:
Ah, maybe I made a mistake. I maybe just call this script instead of using 'source' command. Sorry for the noise. Closing.